client/devdocs/design/index.jsx

```diff
@@ -284,6 +285,7 @@ export default class DesignAssets extends Component {
 					<Wizard readmeFilePath="wizard" />
 					<WizardProgressBar readmeFilePath="wizard-progress-bar" />
 					<WpcomColophon readmeFilePath="wpcom-colophon" />
+					<ListTile readmeFilePath="list-tile" />
```

Comment:
New component looks great in docs page! I just noticed it doesn't render readme part. It looks we need to use absolute path there:

```suggestion
					<ListTile readmeFilePath="/packages/components/src/list-tile" />
```

It would make sense to put it into alphabetical order as we do with all other components here.

Reply:
Nice catch, thank you!